src/components/BlockCombinator/index.js

```diff
@@ -64,15 +94,18 @@ function BlockCombinator() {
           blockTitle ? (
             <Block
               key={`${blockTitle}${index}`}
-              id={`${blockTitle}${index}`}
+              id={`"blocksLogic${index}`}
```

Comment:
```suggestion
              id={`blocksLogic${index}`}
```
id 값에 오타가 들어간걸로 보입니다. 의도하신게 아니라면 삭제해야 될 것 같습니다!